README.md

```diff
@@ -2,7 +2,8 @@
 
 Licensed under the Apache License, Version 2.0. http://www.apache.org/licenses/LICENSE-2.0.txt
 
-Tested with Travis CI.
+Tested with Travis CI. and COVERALLS
```

Comment:
All caps and the period looks a bit weird.  Can we just place the badges right under the heading without any text?  The text seems redundant.